ecaudit/src/main/java/com/ericsson/bss/cassandra/ecaudit/facade/DefaultAuditor.java

```diff
@@ -79,6 +79,9 @@ private boolean shouldAudit(AuditEntry logEntry)
         {
             return !filter.isWhitelisted(logEntry);
         }
+        catch(Exception ex) {
```

Comment:
Everywhere else in the code, Exception variables are called "e"
Please update that to be consistent